.coderabbit.yaml

```
@@ -0,0 +1,208 @@
+    websocket_realtime:
+      pattern: "WebSocket Real-time"
+      description: "Implements real-time updates via WebSocket subscriptions"
+    react_with_vite:
+      pattern: "React with Vite"
+      description: "Uses React with Vite for frontend, not Next.js"
+    sqlite_database:
+      pattern: "SQLite Database"
+      description: "Database layer uses SQLite with better-sqlite3"
+    testing_stack:
+      pattern: "Testing Stack"
+      description: "Testing uses Vitest for unit tests and Playwright for E2E"
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)